etl/steps/data/meadow/irena/2024-10-29/renewable_power_generation_costs.py

```diff
@@ -236,16 +310,20 @@ def combine_global_and_national_data(tb_costs_global: Table, tb_costs_national:
     tb_combined = pr.concat([tb_costs_global, tb_costs_national], ignore_index=True).astype({"year": int})
 
     # Convert from long to wide format.
-    tb_combined = tb_combined.pivot(index=["country", "year"], columns="technology", values="cost").reset_index()
-
-    # Remove name of dummy index.
-    tb_combined.columns.names = [None]
+    tb_combined = tb_combined.pivot(
+        index=["country", "year"], columns="technology", values="cost", join_column_levels_with="_"
+    )
 
-    # Underscore column names.
-    tb_combined = tb_combined.underscore()
+    # Improve table format.
+    tb_combined = tb_combined.format(sort_columns=True)
 
-    # Set an appropriate index and sort conveniently.
-    tb_combined = tb_combined.set_index(["country", "year"], verify_integrity=True).sort_index().sort_index(axis=1)
+    # Add units.
+    for column in tb_combined.columns:
+        tb_combined[column].metadata.unit = f"constant {EXPECTED_DOLLAR_YEAR} US$ per kilowatt-hour"
+        tb_combined[column].metadata.short_unit = "$/kWh"
+        tb_combined[
+            column
+        ].metadata.description_short = "This data is expressed in US dollars per kilowatt-hour. It is adjusted for inflation but does not account for differences in living costs between countries."
```

Comment:
Maybe you can drop the but in this text if such adjustment is not commonly done in the area.

Reply:
I don't think it's bad to clarify it. Unless you think it's better to not mention it, I'd prefer to keep it.

Reply:
Yeah, no strong feelings about it. If you prefer, you can also add this renewable_power_generation_costs